managed/src/main/java/com/yugabyte/yw/controllers/EncryptionAtRestController.java

```diff
@@ -49,35 +50,29 @@ public Result createKMSConfig(UUID customerUUID, String keyProvider) {
                 customerUUID.toString(),
                 keyProvider
         ));
-        try {
-            TaskType taskType = TaskType.CreateKMSConfig;
-            ObjectNode formData = (ObjectNode) request().body().asJson();
-            KMSConfigTaskParams taskParams = new KMSConfigTaskParams();
-            taskParams.kmsProvider = Enum.valueOf(KeyProvider.class, keyProvider);
-            taskParams.providerConfig = formData;
-            taskParams.customerUUID = customerUUID;
-            taskParams.kmsConfigName = formData.get("name").asText();
-            formData.remove("name");
-            UUID taskUUID = commissioner.submit(taskType, taskParams);
-            LOG.info("Submitted create KMS config for {}, task uuid = {}.", customerUUID, taskUUID);
-
-            // Add this task uuid to the user universe.
-            CustomerTask.create(Customer.get(customerUUID),
-                    customerUUID,
-                    taskUUID,
-                    CustomerTask.TargetType.KMSConfiguration,
-                    CustomerTask.TaskType.Create,
-                    taskParams.getName());
-            LOG.info("Saved task uuid " + taskUUID + " in customer tasks table for customer: " +
-                    customerUUID);
-
-            auditService().createAuditEntry(ctx(), request(), formData);
-            return new YWResults.YWTask(taskUUID).asResult();
-        } catch (Exception e) {
-            final String errMsg = "Error caught attempting to create KMS configuration";
-            LOG.error(errMsg, e);
-            return ApiResponse.error(BAD_REQUEST, e.getMessage());
-        }
+        TaskType taskType = TaskType.CreateKMSConfig;
+        ObjectNode formData = (ObjectNode) request().body().asJson();
+        KMSConfigTaskParams taskParams = new KMSConfigTaskParams();
+        taskParams.kmsProvider = Enum.valueOf(KeyProvider.class, keyProvider);
+        taskParams.providerConfig = formData;
+        taskParams.customerUUID = customerUUID;
+        taskParams.kmsConfigName = formData.get("name").asText();
+        formData.remove("name");
+        UUID taskUUID = commissioner.submit(taskType, taskParams);
+        LOG.info("Submitted create KMS config for {}, task uuid = {}.", customerUUID, taskUUID);
+
+        // Add this task uuid to the user universe.
+        CustomerTask.create(Customer.get(customerUUID),
```

Comment:
Customer.getOrBadRequest as first line of this method and use the customer var here.